src/app/components/dev/dev.component.html

```diff
@@ -0,0 +1,42 @@
+<h1 class="govuk-heading-l">Dev components</h1>
+
+<app-tabs>
+  <div *tab="'Filters'">
+    <p class="govuk-body">Filter functionality has following features:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>Single or multi-select against filter types</li>
+      <li>Shows current selection, at the top</li>
+      <li>Search against values</li>
+      <li>Clear specific selected filters</li>
+      <li>Clear entire selected filters/inputs</li>
+    </ul>
+
+    <p class="govuk-body">
+      See <a class="govuk-link" href="/dev/components#input"> bottom of page</a> for example
+      <strong>Input & Output</strong>
+    </p>
+
+    <!-- Filters component -->
+    <app-filters (filterEvent)="getFilters($event)" [filters]="filters"></app-filters>
+
+    <h3 class="govuk-heading-s" id="input" style="padding-top: 40px">Input</h3>
+
+    <p class="govuk-body">
+      This filter component takes in a generic object and renders UI based on that component, see example object below.
+    </p>
+    <pre> {{ filters | json }} </pre>
+
+    <h3 class="govuk-heading-s">Output</h3>
+
+    <p class="govuk-body">
+      This filter component outputs the currently filtered values so they can be used by a parent component, see example
+      object below. To see actual output, click 'Apply filters' button on the filter component.
+    </p>
+    @if (selectedFilters) {
+      <pre> {{ selectedFilters | json }} </pre>
+    } @else {
+      <pre> {{ exampleOutput | json }} </pre>
+    }
+  </div>
```

Comment:
Might be nice to have the JSON preview stuff to the side of the comp?
Scrolling up and down hurts my fingers 😄